cmd/tailscale/cli/get.go

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package cli
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"maps"
+	"reflect"
+	"slices"
+	"strconv"
+	"strings"
+
+	"github.com/peterbourgon/ff/v3/ffcli"
+	"tailscale.com/ipn"
+	"tailscale.com/ipn/ipnstate"
+	"tailscale.com/types/opt"
+)
+
+var getCmd = &ffcli.Command{
+	Name:       "get",
+	ShortUsage: "tailscale get <setting>",
+	ShortHelp:  "Print specified settings",
+	LongHelp: `"tailscale get" prints a specific setting.
+
+Only one setting will be printed.
+
+SETTINGS
+` + getSettings.settings(),
+	FlagSet:   newFlagSet("get"),
```

Comment:
let's add a `--json` flag while we're here for quoted strings and arrays and distinguishing opt.Bool null/false/true etc

Reply:
Initially, I only imagined that `tailscale set --setting=$(tailscale get setting)` would roundtrip to be a no-op, which is what this PR implements.

But this is an interesting suggestion. I’ll write a new PR stacked on top of this one that implements `--json`. @bradfitz , I assume you don’t mean to block this PR to get this feature.

Reply:
I actually intend to block this until until we've thought about it for a bit. 😄

We try not to rush in CLI changes that are hard to undo.

See my recent up/set issue as an example.